cw/capture_sha3_cw310.yaml

```diff
@@ -8,24 +8,25 @@ capture:
   key_len_bytes: 16
   plain_text_len_bytes: 16
   output_len_bytes: 32
+  # Switch the masking on or off - Works for SHA3 only. Doesn't work when
+  # processing key material.
```

Comment:
Is "masking" here referring to the initial sharing of the the input data? i.e. the design is implemented using DOM and the HW PRNG is still enabled.

Reply:
You're right, the design is implemented using DOM. Masking here refers to
- Message masking upon loading into the SHA3 core
- Advancing the PRNG while the SHA3 core is busy.
Let me clarify that in the PR.